tools/svdconv/SVDConv/src/SVDConv_Msgs.cpp

```diff
@@ -204,10 +204,11 @@ const MsgTable SvdConv::msgTable = {
   { "M384", { MsgLevel::LEVEL_ERROR,    CRLF_B,   "Number of PMU Event Counters (found val: '%NUM%') not set or outside range [2..31]. Ignoring PMU entry."} },   // 14.02.2020
   { "M385", { MsgLevel::LEVEL_ERROR,    CRLF_B,   "PMU not supported for CPU '%NAME%'"                                         } },   // 14.02.2020
   { "M386", { MsgLevel::LEVEL_ERROR,    CRLF_B,   "Name not C compliant: '%NAME%' : Brackets [] found"                         } },
-  { "M387", { MsgLevel::LEVEL_ERROR,    CRLF_B,   ""                                                                           } },
+  { "M387", { MsgLevel::LEVEL_ERROR,    CRLF_B,   "CPU: <sauNumRegions> set to null but regions are configured."               } },
```

Comment:
Why are we not using a digit '0' here but a string 'null'. Would this be valid:
```
  <sauNumRegions>null</sauNumRegions>

```

Reply:
fixded. Thought "null" would be better readable.